CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 ## [Unreleased](https://github.com/Kampfkarren/selene/compare/0.25.0...HEAD)
 ### Fixed
 - `string.pack` and `string.unpack` now have proper function signatures in the Lua 5.3 standard library.
+- Moved `math.log` second argument addition from Lua 5.3 std lib to 5.2 std lib
+- Added `table.move`to Lua 5.3 standard library

```

Comment:
The bottom 3 of these should be in Added, not Fixed. Make a new section above Fixed